css/popup.css

```diff
@@ -0,0 +1,60 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Press Start 2P', cursive;
+}
+
+.restart {
+    visibility: hidden;
+    width: 800px;
+    height: 300px;
+    position: absolute;
+    padding: 30px;
+    left: 100px;
+    top: 100px;
+    border-radius: 5px;
+    background-color: #b3ff0f;
+}
+
+.restart h2  {
+    text-align: center;
+    font-size: 2rem;
+}
+
+.restart h1  {
+    text-align: center;
+    font-size: 5rem;
+}
+
+#btnRestart {
```

Comment:
Can a class be used here, id's are not recommended for css selections.

Reply:
The button keeps inheriting the css from the page that its loaded into so i've tagged it to the id as the id is being used by the javascript query selector